specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,9300 @@
+          "type": "string",
+          "readOnly": true,
+          "description": "The error message."
+        },
+        "error": {
+          "$ref": "#/definitions/ErrorModel",
+          "readOnly": true
+        },
+        "runType": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The run type."
+        },
+        "dataSourceType": {
+          "readOnly": true,
+          "$ref": "#/definitions/DataSourceType"
+        }
+      }
+    },
+    "DiscoveryExecutionDetails": {
+      "description": "The discovery execution details.",
+      "type": "object",
+      "properties": {
+        "discoveryStartTime": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The discovery start time."
+        },
+        "discoveryEndTime": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The discovery end time."
+        },
+        "status": {
+          "enum": [
+            "Queued",
+            "Running",
+            "Succeeded",
+            "Failed",
+            "Cancelled",
+            "Delayed",
+            "Throttled",
+            "CompletedWithExceptions",
+            "CompleteWithWarning"
+          ],
+          "type": "string",
+          "description": "The discovery status.",
+          "x-ms-enum": {
+            "name": "ScanStatus",
+            "modelAsString": true
+          }
+        },
+        "statistics": {
+          "type": "object",
+          "description": "The discovery statistics.",
+          "additionalProperties": {
+            "type": "object"
+          }
+        },
+        "isErrorLogAvailable": {
+          "type": "boolean",
+          "description": "Indicates whether the error log is available."
+        }
+      }
+    },
+    "IngestionExecutionDetails": {
+      "description": "The ingestion execution details.",
+      "type": "object",
+      "properties": {
+        "ingestionStartTime": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The ingestion start time."
+        },
+        "ingestionEndTime": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The ingestion end time."
+        },
+        "status": {
+          "enum": [
+            "None",
+            "Succeeded",
+            "Queued",
+            "InProgress",
+            "SourceUnknown",
+            "PartialSucceeded",
+            "Failed",
+            "Canceled",
+            "Canceling"
+          ],
+          "type": "string",
+          "description": "The ingestion status.",
+          "x-ms-enum": {
+            "name": "IngestionPhaseStatus",
+            "modelAsString": true
+          }
+        },
+        "statistics": {
+          "type": "object",
+          "description": "The ingestion statistics.",
+          "additionalProperties": {
+            "type": "object"
+          }
```

Comment:
Another probably erroneous case of `additionalProperties: { type: object }`. And there are two more later in the file. Please correct if appropriate.

Reply:
This one is fixed too